open-company/open-company-web
# Stream/Grid content density

Card: https://trello.com/c/5RvUswLo

Abstract: https://share.goabstract.com/55200285-8f61-43bb-a684-c4802b27d5b8

To test:
- check the new grid view
- check the new stream view
- check the mobile feed
- check the navigation sidebar on desktop
- check the navigation menu on mobile
